src/redux/reducers/experimentSettings/initialState.js

```diff
@@ -4,6 +4,23 @@ export default {
       complete: false,
       stepsDone: new Set([]),
     },
+    dataIntegration: {
+      dataIntegration: {
+        method: 'seuratv4',
+        methodSettings: {
+          seuratv4: {
+            numGenes: 2000,
+            normalisation: 'logNormalise',
+          },
+        },
+      },
+      dimensionalityReduction: {
+        method: 'rpca',
+        numPCs: 30,
+        variationExplained: 91,
+        excludeGeneCategories: [],
+      },
+    },
```

Comment:
Created new schema for storing data integration. This is different from the schema in the database, where previously it was only `dimensionalityReduction`. This has to be taken into account when implementing mechanism for storing this in the database.

Reply:
thanks for raising that, @cosa65 do we have a ticket for updating the API schema about this? If so, we will also need to update this file too: https://github.com/biomage-ltd/api/blob/master/src/api/route-services/mock-data.json